straxen/get_corrections.py

```diff
@@ -37,72 +38,55 @@ def correction_options_wrapped(run_id, conf, *arg):
 
 @export
 @correction_options
-def get_to_pe(run_id, gain_model, n_pmts):
-    if not isinstance(gain_model, tuple):
-        raise ValueError(f"gain_model must be a tuple")
-    if not len(gain_model) == 2:
-        raise ValueError(f"gain_model must have two elements: "
-                         f"the model type and its specific configuration")
-    model_type, model_conf = gain_model
-
-    if model_type == 'disabled':
-        # Somebody messed up
-        raise RuntimeError("Attempt to use a disabled gain model")
-    if model_type == 'CMT_model':
-        if not isinstance(model_conf, tuple) or len(model_conf) != 2:
-            # Raise a value error if the condition is not met. We should have:
-            # ("CMT_model", -> To specify that we want to use the online
-            #                  corrections management tool
-            #   (
-            #   "to_pe_model", -> This is to specify that we want  the gains
-            #   "v1", -> The version of the correction 'v1' is the online version
-            #   )
-            raise ValueError('CMT gain model should be similar to:'
-                             '("CMT_model", ("to_pe_model", "v1"). Instead got:'
-                             f'{model_conf}')
-        # is this the best way to do this?
-        is_nt = n_pmts == straxen.n_tpc_pmts or n_pmts == straxen.n_nveto_pmts or n_pmts == straxen.n_mveto_pmts
-
-        corrections = straxen.CorrectionsManagementServices(is_nt=is_nt)
-        to_pe = corrections.get_corrections_config(run_id, model_conf)
-
-        return to_pe
-
-    elif model_type == 'to_pe_per_run':
-        warn("to_pe_per_run will be replaced by CorrectionsManagementSevices",
-             DeprecationWarning, 2)
-        # Load a npy file specifying a run_id -> to_pe array
-        to_pe_file = model_conf
-        x = straxen.get_resource(to_pe_file, fmt='npy')
+def get_to_pe(run_id, conf, n_pmts):
+
+    if isinstance(conf, str) and conf.startswith('https://raw'):
+        # Legacy support for pax files
+        x = straxen.get_resource(elife_conf, fmt='npy')
```

Comment:
Something went wrong copy pasting things here, `elife_conf` is not defined.

Reply:
fixed